src/github/folderRepositoryManager.ts

```diff
@@ -1961,11 +1961,13 @@ export class FolderRepositoryManager implements vscode.Disposable {
 							if (!this._updateMessageShown) {
 								this._updateMessageShown = true;
 								const pull = 'Pull';
+								const always = 'Always Pull';
 								const never = 'Never Show Again';
 								const result = await vscode.window.showInformationMessage(
 									`There are updates available for pull request ${pr.number}: ${pr.title}.`,
 									{},
 									pull,
+									always,
```

Comment:
Is this order correct?

Reply:
Yes, having it second makes sense to me.